payment-request/PaymentRequestUpdateEvent/updateWith-call-immediate-manual.https.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="https://www.w3.org/TR/payment-request/#updatewith()-method">
+<link rel="help" href="https://github.com/w3c/payment-request/pull/591">
+<title>
+  PaymentRequestUpdateEvent.updateWith() needs to be called immediately
+</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+setup({ explicit_done: true, explicit_timeout: true });
+const applePay = Object.freeze({
```

Comment:
I've always been curious, why are we freezing objects? 🥶

Reply:
It was just to prevent the tests (or people contributing tests) for changing the objects by accident... it was kinda just a safety measure against ourselves.

Reply:
Thank you for the explanation 👍